### PR TITLE
WIP- Docs/docs page updates

### PR DESCRIPTION
**Pull request by TashaBasalt**
Added some content to getting-started.md, config.md, and intro.md

I wasn't sure if I should make a new branch for each file I worked on, but will do so in the future if needed.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1126343145187461/1136460172283743)

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6096,6 +6096,11 @@ chardet@^0.7.0:
   resolved "https://registry.yarnpkg.com/chardet/-/chardet-0.7.0.tgz#90094849f0937f2eedc2425d0d28a9e5f0cbad9e"
   integrity sha512-mT8iDcrh03qDGRRmoA2hmBJnxpllMR+0/0qlzjqZES6NdiWDcZkCNAk4rPFZ9Q85r27unkiNNg8ZOiwZXBHwcA==
 
+charenc@~0.0.1:
+  version "0.0.2"
+  resolved "https://registry.yarnpkg.com/charenc/-/charenc-0.0.2.tgz#c0a1d2f3a7092e03774bfa83f14c0fc5790a8667"
+  integrity sha1-wKHS86cJLgN3S/qD8UwPxXkKhmc=
+
```
Comment (glassdimly) [inline yarn.lock]: It seems like this `lock` file got committed accidentally?

If not, then it should probably be in a separate PR with some justification for adding `charence`,  `crypto` and `md5`. 

Otherwise this looks good!

Comment (brandonreid) [reply yarn.lock]: @TashaBasalt Plz resolve this comment^
